src/style.css

```diff
@@ -1,3 +1,66 @@
-body {
+html {
+  --p-header: 50px;
+  --active-color: #00acdc;
+}
+
+body,
+li,
+ul,
+a {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
```

Comment:
Consider using a relative font size (e.g., `em` or `rem`) instead of a fixed size (`px`) for better accessibility and responsiveness.